<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carico_mag</class>
 <widget class="QDialog" name="carico_mag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>carico_mag</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="8">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Carico magazzino</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Carico magazzino</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Totale acquistato in magazzino</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="tot_car"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="quan"/>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Quantit√† residua in magazzino</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Informazioni dettagliate</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QTextEdit" name="dettagli"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista carico</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="tab_view">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>15</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>9</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ex_pdf">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Esporta pdf</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="ex_csv">
     <property name="text">
      <string>Esporta csv</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/excel.png</normaloff>:/images/excel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="export_excel">
     <property name="text">
      <string>Esporta excel</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/excel.png</normaloff>:/images/excel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pr_st">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Stampa</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/document-print.png</normaloff>:/images/document-print.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="del_rec">
     <property name="text">
      <string>Elimina</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Esci</string>
     </property>
     <property name="icon">
      <iconset resource="../../backup_restore_db/images.qrc">
       <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dettagli</tabstop>
  <tabstop>tot_car</tabstop>
 </tabstops>
 <resources>
  <include location="../../backup_restore_db/images.qrc"/>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
